### PR TITLE
Updated Custom Godlike Mixing NPC

### PR DESCRIPTION
**Pull request by Jeybla**
- Adds SE and TE Godlike Mixings
- Fixes some Mixing Items.
- Sources:
- http://irowiki.org/wiki/God_Items_Quest
- http://irowiki.org/wiki/God_Items_Quest_2
- http://irowiki.org/wiki/God_Items_Quest_3

### CODE REVIEW

**Comment by Lemongrass3110** [comment]
@rathena/rathena-script-db-developers could anyone look into this please?

**Comment by Jeybla** [comment]
Just to be said: It's not important for me. I just used and corrected this npc and liked to share it.
It hasn't been updated since 2012.

File: npc/custom/quests/valhallen.txt [hunk 1]
```diff
@@ -19,21 +24,21 @@ prontera,147,171,5	script	Lenneth	811,{
...
 			mes "[Lenneth]";
 			mes "^3355FFMjolnir^000000 is the mighty Hammer of Thor.";
-			setarray .@Items[0],1530,1531,2,984,20,985,5,969,40,7074,2,7075,4,7078,5,7087,5,7089,5;
+			setarray .@Items[0],1530,1522,2,984,20,985,5,969,40,7074,2,7075,4,7078,5,7091,5,7089,5;
 			break;
 		case 2:
 			mes "[Lenneth]";
 			mes "^3355FFSleipnir^000000 are boots made after Odin's War Horse.";
-			setarray .@Items[0],2410,2406,2,984,1,969,20,985,10,7076,3,7079,5,7083,3,7086,3;
+			setarray .@Items[0],2410,2406,2,984,20,969,20,985,10,7076,3,7079,5,7083,3,7086,4,7087,3;
```
Comment (Atemo) [inline npc/custom/quests/valhallen.txt]: From https://github.com/rathena/rathena/blob/master/npc/quests/seals/god_weapon_creation.txt#L650
and IRO Sleipnir doesn't require 20 Oridecon (**ID: 984**)

File: npc/custom/quests/valhallen.txt [hunk 2]
```diff
@@ -43,26 +48,71 @@ prontera,147,171,5	script	Lenneth	811,{
...
+			break;
+		case 7:
+			mes "[Lenneth]";
+			mes "The ^3355FFBrynhild^000000 is the warrior's armor.";
+			setarray .@Items[0],2383,7830,1,7831,1,7832,1,7833,1,7834,1,7510,100,2357,1,969,10,985,20;
+			break;
+		case 8:
+			mes "[Lenneth]";
+			mes "The ^3355FFHervor^000000 is named after the valkyrie who ensures victory for her warriors.";
+			setarray .@Items[0],2181,6595,2,6596,1,6594,4,6597,3,6602,1,6604,1,6605,1,2115,1;
```
Comment (Atemo) [inline npc/custom/quests/valhallen.txt]: May you add Muspellium and Essence_Of_Rune menu creation (like Gleipnir to make Megingjard) ?
https://github.com/rathena/rathena/blob/woe_te-restrictions/npc/re/quests/woe_te/te_goditem_alde1.txt#L329
https://github.com/rathena/rathena/blob/woe_te-restrictions/npc/re/quests/woe_te/te_goditem_alde1.txt#L349
http://irowiki.org/wiki/God_Items_Quest_3#cite_ref-1

**Comment by Jeybla** [comment]
Thanks. I like the idea to add Muspellium an Essence of Rune!

**Comment by Atemo** [comment]
Thank :+1:

**Pull request merged**